afids_utils/tests/test_afids.py

```diff
@@ -36,12 +36,14 @@ def human_mappings() -> list[dict[str, str]]:
 
 class TestAfidPosition:
     @given(pos=af_st.afid_positions())
-    def test_valid_position(
-        self,
-        pos: AfidPosition,
-    ):
+    def test_valid_position(self, pos: AfidPosition):
         """Just checks that a hypothesis-generated AfidsPosition inits."""
 
+    @given(pos1=af_st.afid_positions(), pos2=af_st.afid_positions())
+    def test_valid_distance(self, pos1: AfidPosition, pos2: AfidPosition):
```

Comment:
I would also check that the result is a tuple of floats

Reply:
This is no longer relevant with the refactored changes - will request a new review once everything has been implemented.